hosts/build02/configuration.nix

```diff
@@ -12,6 +12,9 @@
     inputs.self.nixosModules.emergency-access
   ];
 
+  # the default zpool import services somehow times out while this import works fine?
+  boot.initrd.systemd.services.zfs-import-zroot.serviceConfig.ExecStartPre = "${config.boot.zfs.package}/bin/zpool import -N -f zroot";
```

Comment:
I have the feeling there are still uids somewhere that get picked up by zfs from old pools.... And than it timesout waiting for these ghost devices that no longer exist.
We have some logic in zfs that makes the system wait for all devices to be available, which this preStart by passes. However since we always need both devices to boot anyway, this shouldn't be an issue.

Reply:
Seems we need to use this everywhere now, was testing install on build01 and it wouldn't work without it. https://github.com/nix-community/infra/commit/347fbab61004ce5d9638bf3aa634d1f0ac052f40